### ISSUE TITLE
UnboundLocalError: 'projected' referenced before assignment

### ISSUE THREAD

**Issue by Tele-Pet** [jawensi/DEMto3D-QGIS-Plugin#9]
I'm getting the following error:

```
An error has occured while executing Python code: 

UnboundLocalError: local variable 'projected' referenced before assignment 
Traceback (most recent call last):
  File "/Users/mbp/.qgis2/python/plugins/DEMto3D/DEMto3D_Dialog/DEMto3D_dialog.py", line 130, in do_export
    parameters = self.get_parameters()
  File "/Users/mbp/.qgis2/python/plugins/DEMto3D/DEMto3D_Dialog/DEMto3D_dialog.py", line 500, in get_parameters
    "z_inv": z_inv, "z_base": z_base, "projected": projected, "crs_layer": self.layer.crs(),
UnboundLocalError: local variable 'projected' referenced before assignment
```

**Comment by jawensi** [comment]
Hello which map units are using??
I think the problem is that DEMto3D only works with data in meters and degrees

**Comment by Tele-Pet** [comment]
I believe you are right.  My data has been in feet.  Once I reprojected to a CRS in either degrees or meters, and then the set the project to the CRS of given layer, it was able to output an `.stl` vile.  Thanks for following up!

Also, perhaps I overlooked it, but I don't think I see mention of the degrees/meters requirement in the plugin's details over at the [QGIS plugin page](http://plugins.qgis.org/plugins/DEMto3D/).   Perhaps it'd be helpful to highlight that requirement somewhere?

Thanks again, and looking forward to exploring this plugin further...
